README.md

```diff
@@ -0,0 +1,53 @@
+# EQ Performance Benchmark
+
+## Summary
+
+This is a performance benchmarking tool designed to measure the performance of [EQ Survey Runner](https://github.com/ONSDigital/eq-survey-runner).
+
+This repository was heavily inspired by the [census performance tests](https://github.com/ONSdigital/census-eq-performance-tests)
+
+The benchmark uses locust to perform a load test. The locust test consists of a set of endpoints which are generated dynamically from a HAR file. This allows us run a functional test, or a manual test representing a normal user's behaviour. The HTTP requests from this test can be exported to a HAR file, which can then be imported into this benchmarking tool. The same requests will be modified slightly and rerun during the locust benchmark.
+
+## Instructions
+
+Open the network inspector in Chrome or Firefox and ensure 'preserve log' is ticked. Run your manual or functional test.
+
+After the test is complete, right-click on the network inspector and save the log as a HAR file. Save this file in the root directory of this project as `requests.har` or alternatively use the path to this HAR file in the `HAR_FILEPATH` environment variable when running locust.
+
+To run a benchmark, use:
+
+```
+pipenv install
+pipenv run ./run.sh
+```
+
+This will run 1 minute of locust requests with 1 user and no wait time between requests. It will also process the CSV output to combine `tx_id` in the requests.
+
+The output file can be found in `output_requests_processed.csv`
+
+If you'd like to run locust on its own, you can use the following commands
+
+For the web interface:
+
+`HAR_FILEPATH="requests.har" EQ_ID="census" FORM_TYPE_ID="household" locust -f runner_benchmark/har_replay_test.py --host=http://localhost:5000`
+
+For a command line interface ( with output CSV file ):
+
+`HAR_FILEPATH="requests.har" EQ_ID="census" FORM_TYPE_ID="household" locust -f runner_benchmark/har_replay_test.py --host=http://localhost:5000 --no-web -t 1m -c 100 -r 10 --csv=output`
+
+## Configuration
+
+The following environment variables can be used to configure the locust test:
+
+- `HAR_FILEPATH` - The filepath of the HAR file relative to the base project directory
+- `EQ_ID` - The EQ ID of the questionnaire being tested (Used to generate the JWT)
+- `FORM_TYPE_ID` - The form type of the questionnaire being tested (Used to generate the JWT)
```

Comment:
Says: *following environment variables `can` be used* but would be good to mention that `EQ_ID` and `FORM_TYPE_ID` must be set.

Reply:
Think I'll change this as per Andrew's comment to default to the values from the HAR file.